docs/sources/mimir/send/otel-exponential-histograms/_index.md

```diff
@@ -0,0 +1,117 @@
+{{< /admonition >}}
+
+You can use the OpenTelemetry protocol (OTLP) to send exponential histograms to Grafana Mimir in their existing format, or you can use the Prometheus remote write protocol to send them as Prometheus native histograms. Refer to [Send native histograms to Mimir](https://grafana.com/docs/mimir/<MIMIR_VERSION>/send/native-histograms/).
+
+The OpenTelemetry SDK supports instrumenting applications in multiple languages. Refer to [Language APIs & SDKs](https://opentelemetry.io/docs/languages/) for a complete list.
+
+## Instrument an application with the OpenTelemetry SDK using Go
+
+Use the OpenTelemetry SDK version 1.17.0 or later.
+
+1. Set up the OpenTelemetry Collector to handle your metrics data. This includes setting up your resources, meter provider, meter, instruments, and views. Refer to [Metrics](https://opentelemetry.io/docs/languages/go/instrumentation/#metrics) in the OpenTelemetry SDK documentation for Go.
+1. To aggregate a histogram instrument as an exponential histogram, include the following view:
+
+   ```
+   Aggregation: metric.AggregationBase2ExponentialHistogram{
+   		MaxSize:  160,
+   		MaxScale: 20,
+   	}
+   ```
+
+   For more information about views, refer to [Registering Views](https://opentelemetry.io/docs/languages/go/instrumentation/#registering-views) in the OpenTelemetry SDK documentation for Go. For information about view configuration parameters, refer to [Base2 Exponential Bucket Histogram Aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#base2-exponential-bucket-histogram-aggregation) in the OpenTelemetry Metrics SDK on GitHub.
+
+## Migrate from explicit bucket histograms
+
+To ease the migration process, you can keep the custom bucket definition of an explicit bucket histogram and add a view for the exponential histogram.
+
+1. Start with an existing histogram that uses explicit buckets.
+1. Create a view for the exponential histogram. Assign this view a unique name and include the exponential aggregation. This creates a metric with the assigned name and exponential buckets.
+
+   The following example shows how to create a metric called `request_latency_exp` that uses exponential buckets.
+
+   ```
+   v := sdkmetric.NewView(sdkmetric.Instrument{
+	   	Name: "request_latency",
+	   	Kind: sdkmetric.InstrumentKindHistogram,
+	   }, sdkmetric.Stream{
+		   Name: "request_latency_exp",
+		   Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{MaxSize: 160, NoMinMax: true, MaxScale: 20},
+	   })
+   ```
+
+   For more information about creating a view, refer to [View](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#view) in the OpenTelemetry Metrics SDK.
+
+1. Modify dashboards to use the exponential histogram metrics. Refer to [Visualize native histograms](https://grafana.com/docs/mimir/<MIMIR_VERSION>/visualize/native-histograms/) for more information.
```

Comment:
I can look into this idea for a future PR, especially when this topic goes GA and will have wider readership. We would still need to substitute all of the classic/native details for explicit bucket/exponential, so I'm not sure it would be worth the effort.